components/account/AccountHeader.vue

```diff
@@ -53,7 +53,7 @@ function previewAvatar() {
 async function toggleNotifications() {
   relationship!.notifying = !relationship?.notifying
   try {
-    const newRel = await client.v1.accounts.follow(account.id, { notify: relationship?.notifying })
+    const newRel = await client.v1.accounts.$select(account.id).follow({ notify: relationship?.notifying })
```

Comment:
Why is there a `?` behind `relationship`, while on line 54 it is specifically given?
If `relationship` would be nullish on line 54, we would already run into a runtime error and never reach line 56

Reply:
Ya, line 54 also doesn't need it on the right side. We can do a PR to remove this later as it is unrelated to the upgrade.